ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java

```diff
@@ -24,5 +30,28 @@ public void setupRobot() throws ExtendedMotorFailureException {
         IntakeArm.setup();
         ChallengeBrake.setup();
         HeadingSensor.setup();
+
+        Cluck.publishRConf("Diagnostics", new RConfable() {
+            @Override
+            public Entry[] queryRConf() throws InterruptedException {
+                ArrayList<Entry> entries = new ArrayList<>();
+                entries.add(RConf.title("Diagnostics"));
+                entries.add(RConf.string("Left Drive Encoder: " + DriveCode.leftDriveEncoder.get()));
+                entries.add(RConf.string("Right Drive Encoder: " + DriveCode.rightDriveEncoder.get()));
+                entries.add(RConf.string("Left Portcullis Encoder: " + Portcullis.leftEncoder.get()));
+                entries.add(RConf.string("Right Portcullis Encoder: " + Portcullis.rightEncoder.get()));
+                entries.add(RConf.string("Intake Arm Encoder: " + IntakeArm.encoder.get()));
+                entries.add(RConf.string("Shooter Encoder: " + Shooter.encoder.getEncoderPosition().get()));
+                entries.add(RConf.string("Heading Yaw: " + HeadingSensor.yawAngle.get()));
+                entries.add(RConf.string("Heading Pitch: " + HeadingSensor.pitchAngle.get()));
+                entries.add(RConf.autoRefresh(1000));
+                return entries.toArray(new Entry[entries.size()]);
+            }
+
+            @Override
+            public boolean signalRConf(int field, byte[] data) throws InterruptedException {
+                return false;
```

Comment:
It's perfectly fine to just return false from signalRConf. It just means you won't listen to any click events, which should be fine.